sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -342,6 +342,7 @@ trait CheckAnalysis extends PredicateHelper {
 
           case InsertIntoTable(t, _, _, _, _)
             if !t.isInstanceOf[LeafNode] ||
```

Comment:
I was wondering why not replacing these one-letter values with something more explicit (like `table` in the following `case`) ? I understand that this is not the purpose of the PR, but just interested if it is normal to highlight those "style" things in PR discussion or not.